floydker/src/floydker/render.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+
+import click
+import yaml
+import jinja2
+import os
+import click_log
+import logging
+import copy
+
+logger = logging.getLogger(__name__)
+
+
+class FilesLoader(jinja2.BaseLoader):
+    """
+    FilesLoader takes a list of template files instead of search path
```

Comment:
Yes, I have updated the comment here. The default load will load everything within the searchpath. This custom loader makes it easy to only load jinja templates that we are interested in.